src/com/lftechnology/java/training/dipak/Test.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.dipak;
+
+import java.util.Scanner;
+
+public class Test {

```

Comment:
Provide class level documentation 
Please refer the below link for writing java docs
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html